src/containers/quiz/question-comment/QuestionComment.styles.ts

```diff
@@ -0,0 +1,67 @@
+import { TypographyVariantEnum } from '~/types'
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    boxSizing: 'border-box',
+    padding: 0,
+    width: '100%'
+  },
+  commentIcon: {
+    alignSelf: 'flex-end'
+  },
+  title: {
+    color: 'primary.300',
+    typography: TypographyVariantEnum.H6,
+    fontSize: '16px',
+    marginBottom: '10px'
+  },
+  inputWrapper: {
+    width: '100%',
+    position: 'relative',
+    display: 'flex',
+    alignItems: 'center'
+  },
+  textField: {
+    width: '100%',
+    '& .MuiOutlinedInput-root': {
+      borderRadius: '8px',
+      paddingLeft: '65px',
```

Comment:
Use spacings for `margin`, `padding`, `gap`, `top`, `right`, `bottom`, `left` properties from our theme, please.

```suggestion
      padding: theme.spacing(0, 5.5, 0, 8)
```


<img width="816" alt="Screenshot 2025-01-30 at 11 13 03" src="https://github.com/user-attachments/assets/5010aa09-6286-4ee6-abbc-eeaa20bc53c5" />

Reply:
Thank you for this reminder. Fixed.